rasa/model.py

```diff
@@ -515,3 +513,32 @@ async def update_model_with_new_domain(
     domain = await importer.get_domain()
 
     domain.persist(model_path / DEFAULT_DOMAIN_PATH)
+
+
+def get_model_for_finetuning(
+    previous_model_file: Optional[Union[Path, Text]]
+) -> Optional[Text]:
+    """Gets validated path for model to finetune.
+
+    Args:
+        previous_model_file: Path to model file which should be used for finetuning or
+            a directory in case the latest trained model should be used.
+
+    Returns:
+        Path to model archive. `None` if there is no model.
+    """
+    if Path(previous_model_file).is_dir():
+        logger.debug(
+            f"Trying to load latest model from '{previous_model_file}' for "
+            f"finetuning."
+        )
+        return get_latest_model(previous_model_file)
+
+    if previous_model_file and Path(previous_model_file).is_file():
```

Comment:
Should this check if it is _not_ a file or dir?

Reply:
👍